programmers/20210610/비밀지도.js

```diff
@@ -0,0 +1,14 @@
+function solution(n, arr1, arr2) {
+    const myArray = []
+    for (let i = 0; i < n; i++) {
+        let temp = ""
+        for (let j = 0; j < n; j++) {
+            temp = ((arr1[i] % 2) + (arr2[i] % 2)) + temp;
+            arr1[i] = parseInt(arr1[i] / 2);
+            arr2[i] = parseInt(arr2[i] / 2);
```

Comment:
```suggestion
            [arr1[i], arr2[i]] = [parseInt(arr1[i] / 2), parseInt(arr2[i] / 2)];
```
되게 깔끔하네요... 정규표현식 사용할 수 있다는 생각도 못했는데... 하나 배워갑니다
따로 더 리뷰할 내용은 없어보이고 이 부분만 이렇게 구조분해 할당으로 표현할 수 있을거같아요

Reply:
리뷰하러왔다가, 구조분해 할당 배우고갑니다.